django_admin_shell/tests/test_view.py

```diff
@@ -36,6 +38,13 @@ def setUp(self):
         self.client_auth = Client()
         self.client_auth.login(username=username, password=password)
 
+        snippets = [
```

Comment:
This assignment is not necessary (local reference). You can create the objects as I have suggested below.